client/my-sites/marketing/style.scss

```
@@ -1144,3 +1150,13 @@
 .sharing-buttons__banner {
 	margin-top: -20px;
 }
+
+.service__new-badge.service__new-badge {
+	vertical-align: bottom;
+	background: #dfd1fb;
+	border-radius: 4px;
+	font-weight: 500;
+	font-size: $font-body-extra-small;
+	padding: 2px 10px;
+	margin-left: 5px;
```

Can we use the [CSS Logical Properties](https://www.w3.org/TR/css-logical-1/) which are [RTL Aware](https://hacks.mozilla.org/2015/10/building-rtl-aware-web-apps-websites-part-2/)?

```suggestion
	margin-inline-start: 5px;
```